packages/cactus-plugin-ccmodel-hephaestus/package.json

```diff
@@ -0,0 +1,99 @@
+{
+  "name": "@hyperledger/cactus-plugin-ccmodel-hephaestus",
+  "version": "2.0.0",
+  "description": "A web service plugin that provides management capabilities on cross-chain transactions visualization.",
+  "keywords": [
+    "Hyperledger",
+    "Cactus",
+    "Integration",
+    "Blockchain",
+    "Distributed Ledger Technology"
+  ],
+  "homepage": "https://github.com/hyperledger/cactus#readme",
+  "bugs": {
+    "url": "https://github.com/hyperledger/cactus/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/hyperledger/cactus.git"
```

Comment:
This should point to the current URL in separate org (https://github.com/hyperledger-cacti/cacti/) - applies to all URLs in here.

I think we will have to fix these paths in other packages as well in one separate PR, so this is not critical, nevertheless it's better to change it now in case that other PR is merged before this.

Reply:
@petermetz Is there already a task for fixing url's in cacti packages or can I create one?

Reply:
Updated